src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml

```diff
@@ -10,7 +10,7 @@
         <link rel="stylesheet" href="~/css/site.css" />
     </environment>
     <environment exclude="Development">
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css"
+        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/4.1.1/css/bootstrap.min.css"
               asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
```

Comment:
Did we verify these fallback test properties still work with the new version of Bootstrap?

Reply:
I tried disabling my internet while running it locally, and the page still rendered properly. It did generate a couple of console errors though.

```
(index):9 GET https://ajax.aspnetcdn.com/ajax/bootstrap/4.1.1/css/bootstrap.min.css 0 ()
(index):74 GET https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js 0 ()
(index):77 GET https://ajax.aspnetcdn.com/ajax/bootstrap/4.1.1/bootstrap.min.js 0 ()
(index):1 Failed to find a valid digest in the 'integrity' attribute for resource 'http://localhost:5000/lib/jquery/dist/jquery.min.js' with computed SHA-256 integrity 'T+aPohYXbm0fRYDpJLr+zJ9RmYTswGsahAoIsNiMld4='. The resource has been blocked.
(index):1 Failed to find a valid digest in the 'integrity' attribute for resource 'http://localhost:5000/lib/bootstrap/dist/js/bootstrap.min.js' with computed SHA-256 integrity 'ayd7RDVyn3C1OqFR2+Z12cvIpmN60wSrB/WBz779/sk='. The resource has been blocked.
```

I'm not sure if that's expected behavior when it falls back.